### ISSUE TITLE
Write text with polyline

### ISSUE THREAD

**Issue by PAVANKUSHVAHA** [Leaflet/Leaflet.draw#737]
Is it possible to write text or distance of polyline. I'm drawing polyline and i need to write text for polylines.
i try leaflet.textpath.js but it is not suitable for me, 
How can i draw,something like this image .
![text](https://user-images.githubusercontent.com/25743611/27367829-a7d6ae22-566c-11e7-8506-376d221c2008.png)

**Comment by ddproxy** [comment]
Yes, sorta. There is a label we add to the cursor when drawing for distances. If you need to put that text somewhere else, there's quite a bit of calculation but you can do it within draw events by inspecting the geometry while it's being drawn.
